src/basic/iovec-util.c

```diff
@@ -62,12 +62,12 @@ char* set_iovec_string_field_free(struct iovec *iovec, size_t *n_iovec, const ch
         return x;
 }
 
-void iovec_array_free(struct iovec *iov, size_t n) {
-        if (!iov)
+void iovec_array_free(struct iovec *iovec, size_t n) {
```

Comment:
Nowadays we make such destructors return NULL instead of void.

Reply:
It's used in CLEANUP_ARRAY, which wants a function that returns `void`.